modules/integration/tests-ui-integration/src/test/java/org/wso2/carbon/mb/ui/test/subscriptions/SubscriptionDeleteTestCase.java

```diff
@@ -0,0 +1,412 @@
+     * @throws XPathExpressionException
+     * @throws MalformedURLException
+     */
+    @Override
+    @BeforeClass()
+    public void init() throws AutomationUtilException, XPathExpressionException, IOException {
+
+        super.init();
+
+        //log into broker
+        driver.get(getLoginURL());
+        LoginPage loginPage = new LoginPage(driver);
+        homePage = loginPage.loginAs(getCurrentUserName(), getCurrentPassword());
+    }
+
+
+    /**
+     * 1. Create 3 queue subscribers for queue "subDeleteQueue"
+     * 2. Log into UI and close one subscriptions
+     * 3. Verify subscription close is successful and only 2 other subscription entries are shown
+     * 4. Verify one of subscribers does not get new messages
+     * @throws IOException
+     * @throws XPathExpressionException
+     * @throws JMSException
+     * @throws NamingException
+     * @throws AndesClientException
+     */
+    @Test(groups = {"wso2.mb", "queue"})
+    public void performQueueSubscriptionCloseTestCase() throws IOException, XPathExpressionException,
+            JMSException, NamingException, AndesClientException, AndesClientConfigurationException,
+            CloneNotSupportedException {
+
+        String queueName = "subDeleteQueue";
+        long expectedMessageCount = 1000L;
+        long messageCountToSend = 1000L;
+        // Creating a initial JMS consumer client configuration
+        AndesJMSConsumerClientConfiguration consumerConfig1 =
+                new AndesJMSConsumerClientConfiguration(getAMQPPort(), ExchangeType.QUEUE, queueName);
+        consumerConfig1.setMaximumMessagesToReceived(expectedMessageCount);
+        consumerConfig1.setPrintsPerMessageCount(expectedMessageCount / 10L);
+        consumerConfig1.setRunningDelay(200);
+        consumerConfig1.setAsync(false);
+
+        AndesJMSConsumerClientConfiguration consumerConfig2 = consumerConfig1.clone();
+
+        AndesJMSConsumerClientConfiguration consumerConfig3 = consumerConfig1.clone();
+
+        AndesJMSPublisherClientConfiguration publisherConfig =
+                new AndesJMSPublisherClientConfiguration(getAMQPPort(), ExchangeType.QUEUE, queueName);
+        publisherConfig.setPrintsPerMessageCount(messageCountToSend / 10L);
+        publisherConfig.setNumberOfMessagesToSend(messageCountToSend);
+
+        // Creating consumer clients
+        AndesClient consumerClient1 = new AndesClient(consumerConfig1, true);
+        consumerClient1.startClient();
+
+        AndesClient consumerClient2 = new AndesClient(consumerConfig2, true);
+        consumerClient2.startClient();
+
+        AndesClient consumerClient3 = new AndesClient(consumerConfig3, true);
+        consumerClient3.startClient();
+
+        //start publisher client
+        AndesClient publisherClient = new AndesClient(publisherConfig, true);
+        publisherClient.startClient();
+
+        //Wait until some messages receive
+        try {
+            Thread.sleep(10000);
```

Comment:
The three tests are individual tests. No value addition making it is a constant. There can be chances we may need to update one test.